### ISSUE TITLE
PHP_POST_MAX_SIZE and/or PHP_UPLOAD_MAX_FILESIZE variable ignored

### ISSUE THREAD

**Issue by tzw9373** [mriedmann/humhub-docker#335]
This environment variable appears to be ignored in multiple recent releases, it's overwritten by PHP settings to be 2 MB regardless of value used. Rolling back to 1.11.4 corrected the issue for me.

**Comment by mriedmann** [comment]
I just tested this with the latest 1.15 version and if I understood your report can't reproduce it.

What I tried:

I used the docker-compose.yml and added the 2 mentioned settings to environments
```yaml
version: '3.1'
services:
  humhub:
...
    environment:
      PHP_UPLOAD_MAX_FILESIZE: 66m
      PHP_POST_MAX_SIZE: 66m
      NGINX_CLIENT_MAX_BODY_SIZE: 66m
 ...
 ``` 

Then I ran `docker compose up`, logged in with admin:test and looked at the settings:

![image](https://github.com/mriedmann/humhub-docker/assets/8947111/2311c744-55c8-44fd-8742-81c11621fd20)

It shows "PHP reported a maximum of (upload_max_filesize): 66 MB" which suspects that the php setting is loaded successfully. I set the max upload to 66 MB in the humhub admin panel.

After that I uploaded a 60mb testfile and tried to download it again. It worked, but there is a small problem with this local setup: The upload works properly, but the download has some problems if you are not accessing humhub using https and port 443. This is most likely because of some incompatibilities between nginx/fastcgi and humhub messing-up the link generation/hostname-detection. 

Besides that, I can't reproduce your problem, sorry.

**TL;DR: Have you tried to set all 3 env vars? I also had to increase the file max size in Humhub under Administration > Settings > Advanced > Files > Maximum upload file size (in MB)**

**Comment by mriedmann** [comment]
Ok after setting those correctly it works. So just keep in mind to set them:

In case of a local test-setup it should be this:
```
HUMHUB_PROTO: 'http'
HUMHUB_HOST: 'localhost:8080'
```

**Comment by jvies** [comment]
I think the issue is fixed in master only (cf commit e05758c).
@mriedmann did you tried on latest tag or on master ?

**Comment by mriedmann** [comment]
Oh very good point ... I tested master. Will do a release later. Thx for pointing this out!

**Comment by jvies** [comment]
Any objection to updating the latest to 1.15.2 instead of 1.15.0 before the new tag ?

**Comment by mriedmann** [comment]
done (with update to 1.15.2)
